### ISSUE TITLE
Publish Morganey 0.0.1 to JCenter

### ISSUE THREAD

**Issue by rexim** [morganey-lang/Morganey#261]
Split from #237

**Comment by rexim** [comment]
Blocked by:
- [x] #270 Error on publishing Morganey 0.0.1 on JCenter 
- [x] #271 Cannot resolve hidden-args after using Morganey 0.0.1 from JCenter

**Comment by rexim** [comment]
Superseded by #324
